docs/src/main/sphinx/functions/table.rst

```diff
@@ -69,3 +69,46 @@ You cannot mix the argument conventions in one invocation.
 
 All arguments must be constant expressions, and they can be of any SQL type,
 which is compatible with the declared argument type.
+
+.. _table-functions-pass-through:
+
+Pass-through function
```

Comment:
We need to figure out what we want to call this .. query pass through? .. and use it consistently. @brianzhan and @kasiafi should chime in to determine what we call this  .. also check what we called this in the release notes..  and we should probably use the same term or update the release notes as well

Reply:
Let's just use the name `query`, and elaborate more in the description. The release note says "Add query table function for full query pass-through to the connector."